hawkular-alerts-rest-tests/src/test/resources/standalone-ha.xsl

```diff
@@ -67,6 +67,9 @@
       <replicated-cache name="publish" mode="ASYNC">
         <transaction mode="BATCH"/>
       </replicated-cache>
+      <replicated-cache name="dataIds" mode="ASYNC">
+        <transaction mode="BATCH"/>
+      </replicated-cache>
       <replicated-cache name="schema" mode="SYNC">
```

Comment:
Yes.
This design has the price to initialize the cache on each new node.
Propagating the events across the cluster is something I want to avoid, to not load the traffic too much.
I think that Definitions listeners can be maintained at local node, as definitinos have visibility per all nodes and it's unlikely we may need to propagate such events yet.
Coordinator won't help too much here, because normally jgroups makes the new node the new coordinator, so everytime a new node is joining probability to perform this is high.
In the other hand, we might have some kind of flags but it's the same, it's a new state to maintain.
So I think that the tradeoff could be to update the cache at initialization per node.
It's a cheap price to pay comparing to the previous design.